packages/app/src/Alert.tsx

```diff
@@ -55,28 +55,28 @@ export const SlackChannelForm = ({
 
   return (
     <>
-      {hasSlackWebhooks && (
-        <div className="mt-3">
-          <Form.Label className="text-muted">Slack Webhook</Form.Label>
-          <Form.Select
-            className="bg-black border-0 mb-1 px-3"
-            required
-            id="webhookId"
-            size="sm"
-            {...webhookSelectProps}
-          >
-            {/* Ensure user selects a slack webhook before submitting form */}
-            <option value="" disabled>
-              Select a Slack Webhook
+      <div className="mt-3">
```

Comment:
always showing this select to ensure `required` arg will prevent form from submitting. we can also improve UX and migrate to react-hook-forms later